crypto/evp/evp_key.c

```diff
@@ -56,6 +56,10 @@ int EVP_read_pw_string_min(char *buf, int min, int len, const char *prompt,
     char buff[BUFSIZ];
     UI *ui;
 
+#ifdef OPENSSL_NO_UI_CONSOLE
+    return ret;
+#endif
+
```

Comment:
Please ifdef out the whole function body.

Reply:
> Please ifdef out the whole function body.

Fixed